### DISCUSSION TITLE
Feature request - Separate database servers on cloud

### DISCUSSION THREAD

**Discussion by kasv-nexergroup** [umbraco/Umbraco.Cloud.Issues#895] [Ideas]
### Issue description

To isolate the three environments dev, stage and prod completely, it would be desirable if at least the prod environment could reside in its own database server/elastic pool.

Furthermore, it would be an advantage if different scaling options were offered for the databases  as well as the possibility of being able to obtain various metrics.

**Comment by sajumb** [comment]
Thanks @kasv-nexergroup. The feature request is noted. We have the separation of databases to individual database servers in our internal roadmap already, but we currently have no target date.

**Comment by tcmorris** [comment]
Also interested in this becoming a feature. Having an isolated production environment at the very least would help, dev/staging could be shared if suitable.

**Comment by FinnSandbeck** [comment]
Also interested in this - Agree that isolated production environment is the important part. dev/staging could still be on shared setup without any problems

**Comment by KevenFFW** [comment]
Also intrigued by this - I concur that having an isolated production environment is important. The development/staging phases could still function well on a shared setup without encountering any issues.

**Comment by ksuvec** [comment]
Thank you all for your inputs. They are highly valued and has helped us prioritize our next features better. I can't make any promises on this feature as for now, but we are looking more into it
